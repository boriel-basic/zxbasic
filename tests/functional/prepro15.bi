' Must return:
' PRINT 10;
' PRINT "Hello";
' macro


#define macro(x) PRINT x;

macro(10)
macro("Hello")
macro


