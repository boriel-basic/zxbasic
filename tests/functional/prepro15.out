#line 1 "prepro15.bi"








PRINT 10;
PRINT "Hello";
macro


